dot/parachain/types/bitvec.go

```diff
@@ -47,7 +47,7 @@ func NewBitVec(bits []bool) (BitVec, error) {
 }
 
 // Bits returns all bits in the BitVec as a slice of bools
-func (bv *BitVec) Bits() []bool {
+func (bv *BitVec) Bits() []bool { // skipcq:GO-W1029
```

Comment:
why we need to add the `// skipcq:GO-W1029`?

Reply:
`BitVec` has methods on both pointer and non-pointer receiver types. This is because `MarshalSCALE()` needs a non-pointer receiver, IIUC.

It seems a bit random to me when DeepSource decides to complain about this. But since it did, I put them there.